README.md

```diff
@@ -1,22 +1,24 @@
-# Easy-NMOS Docker Compose Solution - INCOMPLETE INSTRUCTIONS
+# Easy-NMOS Docker Compose Solution
```

Comment:
I agree with @pkeroulas, that even with a few "to dos" at the end of the page, you can lose the "INCOMPLETE INSTRUCTIONS" now, @rhastie :-)

Reply:
Actually, I wonder whether "Docker Compose Solution" is necessary in the title?